tests/test_server_proto.py

```diff
@@ -37,6 +37,7 @@
 class TestServerProto(tb.QueryTestCase):
 
     ISOLATED_METHODS = False
+    SERIALIZED = True
```

Comment:
Was there some flaky test that needs this?

Reply:
The whole test suite is flaky because it uses `ISOLATED_METHODS=False` and so is subject to races with itself, because not all test cases use transactions.